leda/Setup.md

```diff
@@ -18,69 +18,83 @@ limitations under the License.
 
 SPDX-License-Identifier: Apache-2.0
 -->
-# Setting Up a Leda Image
+The components of the Fleet Management blueprint fall into two categories: the *kuksa.val Databroker*,
+*CSV Provider* and *FMS Forwarder* components are supposed to run in the vehicle, whereas the remaining
+components are supposed to run in a (cloud) back end to which the vehicle is connected via (public internet)
+networking infrastructure.
 
-The setup for this FMS scenario assumes a split into components running in the vehicles and components
-running in a (cloud) backend. The following page describes how to configure a Leda instance to become part
-of this FMS setup. More precisely, we deploy the containers for the feedercan, the fms-forwarder,
-and the Kuksa.val databroker with an FMS-specific vehicle model.
+All of the components can be run on a single Docker host as described in the [top level README](../README.md). However, the following sections describe a more realistic deployment scenario in which
+[Eclipse Leda](https://eclipse-leda.github.io/leda/) is used as the vehicle runtime environment.
 
-## Run Leda
-The following steps assume a running instance of Eclipse Leda. For more details see the
-[Eclipse Leda Getting Started](https://eclipse-leda.github.io/leda/docs/general-usage/)
-This guide was tested with release v0.1.0-M1 of Eclipse Leda.
+This guide was tested with release v0.1.0-M2 of Eclipse Leda.
 
-## Backup existing manifestes
-In Leda, there are manifests to manage the execution of containers by Eclipse Kanto. During the next steps we
-will overwrite some of the default manifests (`databroker.json`, `feedercan.json`). Because of that,
-we recommend to backup the existing manifests from `/data/var/containers/mainfests`.
+# Start Back End Components
 
-## Copy manifests to Leda
-To trigger the execution of the required containers for the FMS setup, copy the manifest files from
-`leda/data/var/containers/manifests` in the host to `/data/var/containers/manifests` in Leda:
+The containers for the back end components are run on the local host using Docker Compose:
```

Comment:
We could mention the different available profiles here or reference the description of the Compose file here.

Reply:
FMPOV this should be covered as part of adding a description (in another PR) of how to set it up using Eclipse Hono. In a cloud based setup I would not suggest to connect the vehicle(s) directly to influxdb ...

Reply:
Describing this in another PR sounds good to me, as far as the 'direct' profile is the default.